cmd/cli/kubectl-kyverno/commands/test/test.go

```diff
@@ -98,6 +98,51 @@ func runTest(out io.Writer, testCase test.TestCase, registryAccess bool, auditWa
 	} else {
 		fmt.Fprintln(out, "  Applying", len(policies)+len(validatingAdmissionPolicies), pluralize.Pluralize(len(policies)+len(validatingAdmissionPolicies), "policy", "policies"), "to", len(uniques), pluralize.Pluralize(len(uniques), "resource", "resources"), "...")
 	}
+	rulesmap := make(map[string]string)
```

Comment:
Since this variable holds a mapping of policies to rules, it can be renamed to `policyToUnknownRules`.

What if two rules don't exist in the policy and we need to store both rules? I think we could use a slice of strings for rules. Maybe something like this `map[string][]string`?

Reply:
It will store many variables .
I test on 2 policies and 2 rules name failing.